src/wazuh_modules/syscollector/sysInfoWin.cpp

```diff
@@ -35,14 +40,13 @@ constexpr auto VISTA_REG_HOTFIX{"SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion
 constexpr auto SYSTEM_IDLE_PROCESS_NAME{"System Idle Process"};
 constexpr auto SYSTEM_PROCESS_NAME{"System"};
 
-static bool isVistaOrLater()
+struct CharDeleter
```

Comment:
why do you need this?

Reply:
It's not needed anymore. I'll remove it.